docs/1.4/upgrading.md

```diff
@@ -72,3 +72,52 @@ This class has several unused methods, or methods with an existing alternative:
 The `Converter` class has been deprecated - switch to using `CommonMarkConverter` instead.
 
 The `ConverterInterface` has been deprecated - switch to using `MarkdownConverterInterface` instead.
+
+### `heading_permalink/inner_contents` configuration option
```

Comment:
Let's move this into `docs/1.5/upgrading.md`.

Reply:
Whoops, had both folders open. Done.